app/models/student.rb

```diff
@@ -101,12 +109,18 @@ class Student < ActiveRecord::Base
   validates :exercise_limitations, presence: true
   validates :first_name, presence: true
   validates :gender, presence: true
-  validates :guardian_email, presence: true
-  validates :guardian_first_name, presence: true
-  validates :guardian_last_name, presence: true
-  validates :guardian_phone_number, presence: true
-  validates :guardian_phone_type, presence: true
-  validates :guardian_relationship, presence: true
+  validates :guardian_one_email, presence: true
```

Comment:
If we're adding validations here, should the guardian fields be `not null` on the db side as well?